core/src/main/java/io/questdb/cutlass/line/AbstractLineSender.java

```diff
@@ -136,6 +137,11 @@ public AbstractLineSender field(CharSequence name, boolean value) {
         return this;
     }
 
+    public AbstractLineSender field(CharSequence name, Timestamp value) {
```

Comment:
the `$()` method accepts `long` as a timestamp. I think it would be nice to keep this method consistent. Both this method and `$()` accept a timestamp. The only(?) difference is that `$()` is a designated timetamp. If you feel `long` is too low-level then I think something like `java.time.Instant` might make more sense. wdyt? 

fwiw: there is a new ILP API being created. your feedback would be appreciated! see: https://github.com/questdb/questdb/pull/2184/files#diff-ad4e3741842aebe7c05447934d1bfb1254001b7da3f80472f43502b363ea3082